mountpoint-s3/tests/fuse_tests/cache_test.rs

```diff
@@ -1,23 +1,100 @@
 use crate::common::cache::CacheTestWrapper;
 use crate::common::fuse::create_fuse_session;
 use crate::common::fuse::s3_session::create_crt_client;
-use crate::common::s3::{get_express_bucket, get_standard_bucket, get_test_bucket_and_prefix};
-use mountpoint_s3::data_cache::{DataCache, DiskDataCache, DiskDataCacheConfig, ExpressDataCache};
+#[cfg(all(feature = "s3_tests", feature = "s3express_tests"))]
```

Comment:
I don't like the scattering of `s3express_tests` feature configs around. Can we clean it into one place?